br-webui/index.js

```diff
@@ -828,18 +834,31 @@ function givemav(){
 		procstatus.router = stdout.search("commrouter") < 0 ? "Not Running" : "Process Running" ;
 		procstatus.nmearx = stdout.search("nmearx") < 0 ? "Not Running" : "Process Running" ;
 		procstatus.driver = stdout.search("wldriver") < 0 ? "Not Running" : "Process Running" ;
+		procstatus.pingproxy = stdout.search("pingproxy") < 0 ? "Not Running" : "Process Running" ;
 		
 		logger.log(stdout+stderr);
 
+		// TODO clean this up, set the font colors/messages in one step after flags have been collected
 		// Check if pixhawk is connected or not 
 		var cmd1 = child_process.exec('ls /dev/serial/by-id/usb-3D_Robotics_PX4_FMU_v2.x_0-if00', function (error, stdout, stderr) {
 			// If not show the respective error messages
 			if (error) {
-				if (procstatus.mav.search("Process Running") >= 0) {
-					procstatus.mav = procstatus.mav.fontcolor("green") + status + ("Pixhawk is unplugged").fontcolor("red");
-				} else {
-					procstatus.mav = procstatus.mav.fontcolor("red") + status + ("Pixhawk is unplugged").fontcolor("red");
-				}
+				child_process.exec('ls /dev/serial/by-id/usb-3D_Robotics_PX4_BL_FMU_v2.x_0-if00', function (error, stdout, stderr) {
```

Comment:
wait, why is this necessary ?

Reply:
There is a difference between 'pixhawk is unplugged' and 'pixhawk is in bootloader'. The PX4 bootloader uses a different usb id than the autopilot program.